packages/ds-healthcare-gov/src/components/ThirdPartyExternalLink/ThirdPartyExternalLink.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 import { ThirdPartyExternalLink } from '@cmsgov/design-system';
```

Comment:
We held back on removing these React imports because of a problem we found somewhere with backwards compatibility. It doesn't look like there's any problem with removing them from the story files, but there may be an issue with removing it from the ones we export in our package. I need to do a little more digging.

Reply:
I noticed that as well with React imports in the main code (not Storybook). I think it probably has something to do with the new JSX transform introduced a few years ago. It's possible the current build setup may be using the old way, which required React in scope even if it wasn't being explicitly used. 

https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reply:
I just checked and yes, it's using the old JSX transform. Since the min peerDependency for React is 17.0.0 it should be possible to upgrade without a major version bump. Several of the build dependencies are almost 2 years old so it might be a good idea to freshen them up as well. If you want I can take a look after I look into the react-day-picker+date-utils thing.

Reply:
I actually did not do a good job of making sure that peer dependency version stayed where it needed to. We're technically still supporting React 16 projects, mostly on the medicare side. Hopefully this year we'll be able to drop React 16. For that reason, I don't think we're ready for this. Thank you for the offer to look into it. If you really want to, it might be good to figure out everything that we will need to and get to change when we do finally drop React 16, even if we can't merge a PR right now.

Reply:
For the ones that I removed, there was no JSX being rendered in the file so React wasn't needed regardless. It looks like the medicare.gov home page is on 17.0.2, according to React devtools? I used to have visibility into the medicare and healthcare repos but they moved them late last year.

Reply:
Oh, I understand. I didn't realize that there was no JSX in those files. Carry on!